cellxgene_schema_cli/tests/fixtures/examples_validate.py

```diff
@@ -190,6 +296,36 @@
     "tissue_type_colors": numpy.array(["black", "pink"]),
 }
 
+visium_library_id = "Proj2023_Lung_C001"
+
+good_uns_with_visium_spatial = {
+    "title": "A title",
+    "default_embedding": "X_umap",
+    "X_approximate_distribution": "normal",
+    "batch_condition": ["is_primary_data"],
+    "spatial": {
+        "is_single": numpy.bool_(True),
+        visium_library_id: {
+            "images": {
+                "hires": numpy.array([[[1, 2, 3], [4, 5, 6]], [[7, 8, 9], [10, 11, 12]]]),
+                "fullres": numpy.array([[[1, 2, 3], [4, 5, 6]], [[7, 8, 9], [10, 11, 12]]]),
+            },
+            "scalefactors": {
+                "spot_diameter_fullres": 1.0,
+                "tissue_hires_scalef": 1.0,
+            },
+        },
+    },
+}
+
+good_uns_with_slide_seqV2_spatial = {
+    "title": "A title",
+    "default_embedding": "X_umap",
+    "X_approximate_distribution": "normal",
+    "batch_condition": ["is_primary_data"],
+    "spatial": {"is_single": numpy.bool_(True)},
```

Comment:
let's make "is_single" a non-numpy boolean True here, since both should be valid. Keep the numpy bool in the visium uns so we can cover both cases